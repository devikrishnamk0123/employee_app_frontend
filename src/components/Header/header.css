.top-bar{

    height: 77px;

    background-color: white;

  }

