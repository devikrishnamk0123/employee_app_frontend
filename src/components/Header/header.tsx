import React from 'react';
import './header.css';

const Header: React.FC = () => {
  return <div className='top-bar' data-testid='test for header'></div>;
};

export default Header;
