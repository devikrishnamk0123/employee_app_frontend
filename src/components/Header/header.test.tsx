import { render, screen } from '@testing-library/react';
import Header from './header';

describe('If header works properly', () => {
  test('If header rendered properly', () => {
    render(<Header />);
    const element = screen.getByTestId('test for header');

    expect(element).toMatchSnapshot();
  });
});
