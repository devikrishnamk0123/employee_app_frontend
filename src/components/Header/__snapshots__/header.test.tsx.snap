// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`If header works properly If header rendered properly 1`] = `
<div
  class="top-bar"
  data-testid="test for header"
/>
`;
